Packs/SIGNL4/Integrations/SIGNL4/SIGNL4.py

```diff
@@ -0,0 +1,95 @@
+from CommonServerPython import *
+
+
+class Client(BaseClient):
+
+    def send_signl4_alert(self, **kwargs):
+
+        payload = {
+            "Title": kwargs.get('json_data').get('title'),
+            "Message": kwargs.get('json_data').get('message'),
+            "X-S4-ExternalID": kwargs.get('json_data').get('s4_external_id'),
+            "X-S4-Status": "new",
+            "X-S4-Service": kwargs.get('json_data').get('s4_service'),
+            "X-S4-Location": kwargs.get('json_data').get('s4_location'),
+            "X-S4-AlertingScenario": kwargs.get('json_data').get('s4_alerting_scenario'),
+            "X-S4-Filtering": kwargs.get('json_data').get('s4_filtering'),
+            "X-S4-SourceSystem": "CortexXSOAR"
+        }
+
+        return self._http_request(method='POST', json_data=payload)
```

Comment:
You send the method as an argument to the function and actually define it here. Another reason not to use kwargs here.

Reply:
Yes, you are right.